### PR TITLE
ret, echo implementation

### PR DESCRIPTION
**Pull request by TerryW-tx**
implement ret, echo functions to successfully run example_yunit.gox
use `gop run .` under yap\ytest\demo\example

### CODE REVIEW

File: ytest/classfile.go [hunk 1]
```diff
@@ -99,3 +103,49 @@ func Oauth2(auth string) RTComposer {
 }
 
 // -----------------------------------------------------------------------------
+
+func Echo__0(content string) {
```
Comment (xushiwei) [inline ytest/classfile.go:107]: `echo` already exists as builtin func.
don't need implement it.

File: ytest/response.go [hunk 2]
```diff
@@ -17,19 +17,24 @@
...
+	"io"
+	"io/ioutil"
 	"log"
 	"net/http"
 )
 
 type Response struct {
 	code   int
 	header http.Header
+	body   io.Reader
```
Comment (xushiwei) [inline ytest/response.go:30]: can't meet requirement if use `io.Reader` (can't call `match` multiple times)

Comment (xushiwei) [reply ytest/response.go:30]: And don't add too many features in a pull request.
Can create a single pr for a complex feature.

Comment (TerryW-tx) [reply ytest/response.go]: I'm new to PR workflow. Thanks for your comments. They are significant to my future work. I'll consider adding redundant features in another PR.

**Pull request closed without merging**